src/physical_object.ts

```diff
@@ -169,9 +168,7 @@ export default class PhysicalObject extends GameElement {
                         ctx, camera_position);
     });
 
-    if (!this.is_ground) {
-      this._draw_circle(this.center_of_mass, 1, 2, "black", ctx, camera_position);
-    }
+    this._draw_circle(this.center_of_mass, 1, 2, "black", ctx, camera_position);
```

Comment:
Not sure if the ground has the center of mass calculated? If not, you can initialize it to Vector2D.empty as a dummy value to get this working.

Reply:
Would this code even get called for a `Ground` object? It overrides `draw`.